tests/Tests.ServiceProvider/CustomAutoDataAttributes.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Tests;
+
+[AttributeUsage(AttributeTargets.Method)]
+public class CustomAutoDataAttribute : AutoDataAttribute
+{
+    public CustomAutoDataAttribute() : base(FixtureHelpers.CreateFixture)
+    {
+    }
+}
+
+[AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+public class CustomInlineAutoDataAttribute : InlineAutoDataAttribute
+{
+    public CustomInlineAutoDataAttribute(params object[] args) : base(FixtureHelpers.CreateFixture, args)
+    {
+    }
+}
+
+internal static class FixtureHelpers
+{
+    public static IFixture CreateFixture()
```

Comment:
I've seen you use static factory functions before to encapsulate the fixture creation. I suppose it's a personal preference.
IMO customizations offer more room for reuse than a static factory function.